### DISCUSSION TITLE
What does dual-licenced under Cecill-B V1.0 and MIT mean?

### DISCUSSION THREAD

**Discussion by helenkwok** [iTowns/itowns#2241] [Q&A] [answered]
I am very new to multiple licenses in the same repository. I understand that the contributing codes have to be dual-licenced under Cecill-B V1.0 and MIT. How about using iTowns in other projects? Do I also need to make every one of them open-source and dual-licenced under Cecill-B V1.0 and MIT?

**Comment by jailln** [comment, marked as answer]
Hi,

If you do not modify itowns code, you just need to mention that you use itowns and reproduce the licences (Cecill-B and MIT). If you modify itowns code, in addition to reproducing the licenses, you need to indicate that your new software has been created from on itowns: in the documentation, in a text easily accessible from the interface and on the web site of the new software.

However, these licences have no impact on the licences that you can choose for your project, meaning you do not have to make your project open-source or and dual-licenced under Cecill-B V1.0 and MIT if you use itowns.

If you end up making a project with itowns, don't hesitate to share a demo :)

**Reply by helenkwok** [reply]
Thank you very much for the detailed explanation!
